Testcontainers Support in WireMock Python (#72)
wiremock/python-wiremock: docs/testcontainers.md

> # Python WireMock Test Containers

Reviewer: ```suggestion
# WireMock module for Testcontainers Python
```

> Python wiremock ships with support for [python-testcontainers](https://github.com/testcontainers/testcontainers-python) to eaily start wiremock server from your test suite using python.

Reviewer: ```suggestion
Python WireMock ships with support for [testcontainers-wiremock](https://github.com/testcontainers/testcontainers-python) to easily start WireMock server from your test suite using Python.
```

## Using the context manager

> The simplest way to integrate the wirmock container into your test suite is to use the `wiremock_container` context manager. For pytest users this can be

Reviewer: ```suggestion
The simplest way to integrate the WireMock container into your test suite is to use the `wiremock_container` context manager. For pytest users this can be
```

of options.

## Using the WireMockContainer directly

You can also instantiate the container instance yourself using `WireMockContainer`. The container itself provides methods for creating mapping files and stubs on the container instance which can be used as an alternative
if you maintain your request and response stubs as files.

```python
WireMockContainer(verify_ssl_certs=False)
    .with_mapping(
        "hello-world.json",
        {
            "request": {"method": "GET", "url": "/hello"},
            "response": {"status": 200, "body": "hello"},
        },
    )
    .with_mapping(
        "hello-world-file.json",
        {
            "request": {"method": "GET", "url": "/hello2"},
            "response": {"status": 200, "bodyFileName": "hello.json"},
        },
    )
    .with_file("hello.json", {"message": "Hello World !"})
    .with_cli_arg("--verbose", "")
    .with_cli_arg("--root-dir", "/home/wiremock")
    .with_env("JAVA_OPTS", "-Djava.net.preferIPv4Stack=true")
)
```

> ## Using WireMockContainer inside docker (dind)

Reviewer: ```suggestion
## Using WireMockContainer inside Docker (dind)
```

> It's common that you might need to start testcontainers from inside of another comntainer. The example project in [Test Containers Example](example/docker-compose.yaml) actually does this.

Reviewer: ```suggestion
It's common that you might need to start Testcontainers from inside of another container. The example project in [Testcontainer Example](example/docker-compose.yaml) actually does this.
```
